ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2022.3.22f1
-m_EditorVersionWithRevision: 2022.3.22f1 (887be4894c44)
+m_EditorVersion: 2022.3.23f1
```

Comment:
This is a very minor point, but please make sure that the Git hooks (`./generateHook.sh`) are properly installed. I'm mentioning this because the project version was updated without the README.md being updated along with it—usually, the git hooks take care of this (along with updating `axivion-jenkins.bat`) automatically.

Reply:
Does this mean that every developer needs to run this script once he/she cloned the repository for the first time?

By the way, I made my changes in a git repository that I created with `git worktree`, in which case there is no local `.git` folder in the folder where the commits are made.

Reply:
> Does this mean that every developer needs to run this script once he/she cloned the repository for the first time?

Yes, `./generateHook.sh` should be run whenever the repository is first cloned. Unfortunately, this can't be automated (for security reasons) and hence has to be done manually.